changelogs/unreleased/3092-jenting

```diff
@@ -0,0 +1 @@
+feat: support configure BSR CR to indicate which one is the default
```

Comment:
1) I think you mean BSL instead of BSR?
2) Could you spell it out completely (BackupStorageLocation)?

Reply:
I've updated it, please take a look.